addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/hardware/callbacks/FritzAhaXMLCallback.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.avmfritz.internal.hardware.callbacks;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+
+import org.openhab.binding.avmfritz.internal.ahamodel.DevicelistModel;
+import org.openhab.binding.avmfritz.internal.hardware.FritzahaWebInterface;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Callback implementation for a xml response. Supports reauthorization.
+ * 
+ * @author Christoph Weitkamp
+ * 
+ */
+public class FritzAhaXMLCallback extends FritzAhaReauthCallback {
+    /**
+     * JAXBContext
+     */
+    public static final JAXBContext jaxbContext = initContext();
+
+    /**
+     * 
+     * @return
+     */
+    private static JAXBContext initContext() {
+        JAXBContext jaxbc = null;
+        try {
+            jaxbc = JAXBContext.newInstance(DevicelistModel.class);
+        } catch (JAXBException e) {
+            LoggerFactory.getLogger(FritzAhaXMLCallback.class).error("{}", e.getMessage(), e);

```

Comment:
I cannot fetch the logger only once here. It is used in a `static` method. This is the way to go in this case.